django_project/first_app/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('namehello/<str:name>', views.hello_by_name, name='HelloByName'),
```

Comment:
`/` в конце не поставили
```suggestion
    path('namehello/<str:name>/', views.hello_by_name, name='HelloByName'),
```